src/mixxxapplication.cpp

```diff
@@ -61,6 +61,8 @@ class QMouseEventEditable : public QMouseEvent {
 #endif
 };
 
+constexpr mixxx::Duration kEventNotifyExecTimeWarningThreshold = mixxx::Duration::fromMillis(20);
```

Comment:
What is the reasoning behind this 20 ms? Can you add a comment? A Waveform cycle is 1/60 s = 16.6 ms

Reply:
I adjusted the threshold to 5ms. This prints more messages on my system, but I think all of them are indicating visible performance issues.
After reducing the limit I see many warnings about slow processing of paint events in KnobComposed widgets. But these disapear, if I apply the fix from #12904 . So I think these are real and no false positives.

Reply:
I adjusted this again, to 10ms, as there are still many events in the 5ms range after fixing the pixmap cache.